scripts/ci/build-installers.sh

```diff
@@ -48,8 +48,8 @@ if [ "$ARGV_OPERATING_SYSTEM" == "linux" ]; then
   ./scripts/build/docker/run-command.sh \
     -r "$TARGET_ARCH" \
     -s "$(pwd)" \
-    -c 'make installers-all'
+    -c 'make electron-develop installers-all'
```

Comment:
So as this PR has now been merged, and #1514 apparently fixes this problem, after #1514 has been rebased to `master`, should it also revert `-c 'make electron-develop installers-all'` back to `-c 'make installers-all'` ?